lib/rspec/expectations/failure_aggregator.rb

```diff
@@ -52,7 +52,7 @@ def call(failure, options)
 
     private
 
-      if RSpec::Support::Ruby.jruby? && RSpec::Support::Ruby.jruby_version < '9.2.0.0'
+      if RSpec::Support::Ruby.jruby? && JRUBY_VERSION < '9.2.0.0'
```

Comment:
Revert this, we're keeping the original comparable check because it works correctly with the JRuby strings.